test/test_functions.jl

```diff
@@ -0,0 +1,44 @@
+# Various functions that are useful for running the tests, and used across several test sets.
+
+
+# Initial Condition/Parameter Generators ###
```

Comment:
These are hopefully self-explanatory? The int ones are needed (or useful) in certain cases (e.g. I think in some Hill function cases they are good to have).